packages/code-du-travail-ui/rollup.config.js

```diff
@@ -14,11 +15,25 @@ export default {
       format: "esm"
     }
   ],
-  external: ["react", "prop-types", "styled-components"],
+  external: ["prop-types", "react", "styled-components"],
   plugins: [
     resolve({
-      jsnext: true,
-      main: true
+      browser: true
+    }),
+    commonjs({
+      include: /node_modules\//,
+      // This is tricky, if we ever decide to extract this subrepo from the mono repo,
+      // then these paths will break and we will have to change them
+      namedExports: {
```

Comment:
Bon c'est moche mais c'est le seul moyen que j'ai trouvé pour garder tout ça dans le bundle (i.e. ne pas inclure divers truc dans le tableau "external" plus haut)

Reply:
ᕕ( ᐛ )ᕗ

Reply:
Haha archi bon ! Bordel je viens de voir le `jsnext:main` dans le package.json de l'accordion, je comprenais rien à ce qu'il se passait ! J'étais vraiment en mode https://gph.is/1JIIWIM Tu gères 👏